src/App/Resources/AppResources.resx

```diff
@@ -2275,4 +2275,46 @@
   <data name="AreYouSureYouWantToEnableScreenCapture" xml:space="preserve">
     <value>Are you sure you want to enable Screen Capture?</value>
   </data>
+  <data name="LogInRequested" xml:space="preserve">
+    <value>Log in requested</value>
+  </data>
+  <data name="AreYouTryingToLogIn" xml:space="preserve">
+    <value>Are you trying to log in?</value>
+  </data>
+  <data name="LogInAttemptByXOnY" xml:space="preserve">
+    <value>Log in attempt by {0} on {1}</value>
+  </data>
+  <data name="FingerprintPhrase" xml:space="preserve">
+    <value>Fingerprint phrase</value>
+  </data>
+  <data name="DeviceType" xml:space="preserve">
+    <value>Device Type</value>
+  </data>
+  <data name="IpAddress" xml:space="preserve">
+    <value>IP Address</value>
+  </data>
+  <data name="Time" xml:space="preserve">
+    <value>Time</value>
+  </data>
+  <data name="Near" xml:space="preserve">
+    <value>Near</value>
+  </data>
+  <data name="ConfirmLogIn" xml:space="preserve">
+    <value>Confirm Log In</value>
+  </data>
+  <data name="DenyLogIn" xml:space="preserve">
+    <value>Deny Log In</value>
+  </data>
+  <data name="JustNow" xml:space="preserve">
+    <value>Just Now</value>
+  </data>
+  <data name="MinutesAgo" xml:space="preserve">
+    <value>{0} minutes ago</value>
+  </data>
```

Comment:
To keep the format and better guess that it takes a parameter, please add X as a prefix in the name, i.e. `<data name="XMinutesAgo" xml:space="preserve">`